### ISSUE TITLE
Error codes in Chinese

### ISSUE THREAD

**Issue by Unknown-Tokyo** [Pokecube-Development/Pokecube-Issues-and-Wiki#873]
#### Issue Description:
error code like a “?” in a diamond

#### What happens:
error code in where it should be Chinese characters

#### What you expected to happen:
normal Chinese characters Chinese characters

#### Steps to reproduce:

1.China and Chinese system
2.Open the game, the guidebook which pokewatch give are error codes
...

____
#### Affected Versions (Do *not* use "latest"): Replace with a list of all mods you have in.

- Pokecube AIO: 31.4.1
- Minecraft: 1.16.5
- Forge: 36.2.23

![QQ截图20220123221841](https://user-images.githubusercontent.com/41478553/150682920-efc57c5f-2ba5-4eac-acb8-583e7c2ad21f.png)

**Comment by Thutmose** [comment]
the pokemob's names are normal, try right clicking them with a pokewatch.

I think we are missing a chinese translation for most of the newer stuff as well, since things changed from .lang files to .json files, I don't think the translations were updated :(

**Comment by Sa-ku-mo** [comment]
> 
Current localized guide, rule and moves file are enough to newbee, if the code can normally show, I guess :D.
I want do some work for it, just some .json right?

**Comment by Sa-ku-mo** [comment]
> the pokemob's names are normal, try right clicking them with a pokewatch.
> 
> I think we are missing a chinese translation for most of the newer stuff as well, since things changed from .lang files to .json files, I don't think the translations were updated :(

I have finished some localization files with new .json file, but it can't be properly read with UTF-8 or GB2312 encode, seems it need a fix in code level. The guide book in /data is GB2312.
updated: I tried to pack it as a resource pack. Anyway, it works, but I'm not sure some part of it, and when I just translate one file according to the en_us, something in same category not included, like some wooden things with berry wood.
Should I upload every localization with single pull requests, or upload them in pile?
